docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>we don't need to download the objects, you can pass the s3 url directly into read_csv</p><p><code>df = pd.read_csv("s3://nasa-maap-data-store/file-staging/nasa-map/GEDI_CalVal_Field_Data___2/gedicalval_treedata_usa_neonyell_20200924_r03.csv")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>